src/Api/todos.tsx

```diff
@@ -0,0 +1,25 @@
+import { NewTodo, Todo } from '../types/Todo';
+import { client } from '../utils/fetchclient';
+
+// This function let us take data from api
+export const getTodosByUserId = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+// This function let us delete TODO from api
```

Comment:
Consider using more descriptive comments for functions. For example: 'Deletes a todo with the specified ID from the API.'